libc-test/build.rs

```diff
@@ -954,6 +955,15 @@ fn test_solarish(target: &str) {
         }
     });
 
+    cfg.skip_type(move |ty| match ty {
+        "sighandler_t"
+        | "lgrp_content_t"
+        | "lgrp_lat_between_t"
+        | "lgrp_mem_size_flag_t"
+        | "lgrp_view_t" => true,
```

Comment:
It would be worth getting to the bottom of why these fail the tests.  Are all other enums defined in rust-libc ignored this why?  If not, why do we require them to be skipped here?

Reply:
not sure we have the choice since libc see the size as 0.

Reply:
Which libc?  In C, their `sizeof` evaluates to 4 like normal `int` types.

Reply:
I meant the crate..

`RUNNING ALL TESTS
bad lgrp_content_t signed: rust: 1 (0x1) != c 0 (0x0)
bad lgrp_lat_between_t signed: rust: 1 (0x1) != c 0 (0x0)
bad lgrp_mem_size_flag_t signed: rust: 1 (0x1) != c 0 (0x0)
bad lgrp_view_t signed: rust: 1 (0x1) != c 0 (0x0)`

Reply:
I think what the error is saying here is that there is a signed/unsigned mismatch between the Rust type and the C type. Are C enums unsigned on solaris?

Reply:
sorry my mistake seems they are when I change the signedness.

Reply:
can you retry pls @pfmooney ?